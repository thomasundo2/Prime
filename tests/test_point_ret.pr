pt retPoint(pt a)
{
    printpt(a);
    return a;
}

int main()
{
    curve mycurve;
    pt b;
    mycurve = [(1l, 2l) : 2l];
    b = [0l, 1l] & mycurve;
    printpt(retPoint(b));
    return 0;
}