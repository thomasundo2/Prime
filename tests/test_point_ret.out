[0l, 1l] & [(1l, 2l) : 2l]
[0l, 1l] & [(1l, 2l) : 2l]
